catalog/build_ci/B002.md

```diff
@@ -1,8 +1,8 @@
 ### B002
 
 |**Code**           | **B002** |
-|**Title**          | Automated Code Style Checking. |
-|**Description**    | |
+|**Title**          | Select and Enforce Code Style for Main Programming Languages. |
```

Comment:
```suggestion
|**Title**          | Code Style Checking for Main Programming Languages. |
```
I prefer the item title not be a whole sentence as it was before.

Reply:
I disagree with the suggestion since the previous seems more clear and comprehensive, IMHO.

Reply:
As a compromise, I suggest this if you agree:
```suggestion
|**Title**          | Enforcing a Specific Code Style for Main Programming Languages. |
```

Reply:
I replaced the title with @emranbm suggestion.